app/pktgen-main.c

```diff
@@ -561,6 +574,13 @@ main(int argc, char **argv)
     }
 
     pktgen_log_info("=== Run CLI\n");
+
+    /* Unblock SIGWINCH so main thread
+     * can handle screen resizes */
+    sigemptyset(&set);
+    sigaddset(&set, SIGWINCH);
+    pthread_sigmask(SIG_BLOCK, &set, NULL);
```

Comment:
I just noticed this code looks exactly like the code at line 440, but this one states unblock and the other states block. Should line 582 use SIG_UNBLOCK instead?

Reply:
Indeed, thank you.